Source/PrivateHeaders/Ably/ARTResumeRequestResponse.h

```diff
@@ -0,0 +1,72 @@
+@import Foundation;
+
+@class ARTProtocolMessage;
+@class ARTErrorInfo;
+@protocol ARTErrorChecker;
+
+/**
+ The type of the Realtime system’s response to a resume request, as described by RTN15c.
+ */
+typedef NS_ENUM(NSUInteger, ARTResumeRequestResponseType) {
+    /**
+     RTN15c6: "This indicates that the resume attempt was valid. The client library should move all channels that were in the @ATTACHING@, @ATTACHED@, or @SUSPENDED@ states to the @ATTACHING@ state, and initiate an @RTL4c@ attach sequence for each. The connection should also process any messages queued per @RTL6c2@ (there is no need to wait for the attaches to finish before processing queued messages)."
+     */
+    ARTResumeRequestResponseTypeValid,
+
+    /**
+     RTN15c7: "In this case, the resume was invalid, and the error indicates the cause. The @error@ should be set as the @reason@ in the @CONNECTED@ event, and as the @Connection#errorReason@. The internal @msgSerial@ counter should be reset so that the first message published to Ably will contain a @msgSerial@ of @0@. The rest of the process is the same as for @RTN16c6@: The client library should move all channels that were in the @ATTACHING@, @ATTACHED@, or @SUSPENDED@ states to the @ATTACHING@ state, and initiate an @RTL4c@ attach sequence for each. The connection should also process any messages queued per @RTL6c2@."
```

Comment:
```suggestion
     RTN15c7: "In this case, the resume was invalid, and the error indicates the cause. The `error` should be set as the `reason` in the `ARTRealtimeConnectionState.ARTRealtimeConnected` event, and as the `ARTConnection.errorReason`. The internal `msgSerial` counter should be reset so that the first message published to Ably will contain a `msgSerial` of 0. The rest of the process is the same as for RTN16c6: The client library should move all channels that were in the `ARTRealtimeChannelAttaching`, `ARTRealtimeChannelAttached`, or `ARTRealtimeChannelSuspended` states to the `ARTRealtimeChannelAttaching` state, and initiate an RTL4c attach sequence for each. The connection should also process any messages queued per RTL6c2."
```

Reply:
As above.